modules/ducktests/tests/ignitetest/tests/benchmarks/add_node_rebalance_test.py

```diff
@@ -0,0 +1,94 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import time
+
+from ducktape.mark import parametrize
+from ducktape.mark.resource import cluster
+
+from ignitetest.services.ignite import IgniteService
+from ignitetest.services.ignite_app import IgniteApplicationService
+from ignitetest.tests.utils.ignite_test import IgniteTest
+from ignitetest.version import DEV_BRANCH, IgniteVersion, V_2_8_0
+
+
+class AddNodeRebalanceTest(IgniteTest):
+    NUM_NODES = 4
+    PRELOAD_TIMEOUT = 60
+    DATA_AMOUNT = 1000000
+    REBALANCE_TIMEOUT = 60
+
+    """
+    Test performs rebalance tests.
+    """
+
+    @staticmethod
+    def properties(client_mode="false"):
+        return """
+            <property name="clientMode" value="{client_mode}"/>
+        """.format(client_mode=client_mode)
+
+    def __init__(self, test_context):
+        super(AddNodeRebalanceTest, self).__init__(test_context=test_context)
+
+    def setUp(self):
+        pass
+
+    def teardown(self):
+        pass
+
+    @cluster(num_nodes=NUM_NODES + 1)
+    @parametrize(version=str(DEV_BRANCH))
+    @parametrize(version=str(V_2_8_0))
```

Comment:
Versions should be moved to config. Adding new versions (or ISP versions) will lead to change multiple files just to update version.

Reply:
We want to build tests as stable as we can. 
So the explicit version in test parameters is a thoughtful choice.
Kafka uses approach with the `LATEST_2_8` or `LATEST_2_9` to change all tests to the new minor versions.

But with every major version, one should update tests.

Reply:
Agree with @nizhikov.
Fixed code to use Latest.